collectors/APICalls/breakpointScript.template.js

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable no-undef */
+const stack = (new Error()).stack;
```

Comment:
sorry, one more idea 😬  Have you tried using `console.trace`? Is that better in any way?